doc/prop.xml

```diff
@@ -131,6 +131,40 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="IsPreorderDigraph">
+<ManSection>
+  <Prop Name="IsPreorderDigraph" Arg="digraph"/>
+  <Prop Name="IsQuasiorderDigraph" Arg="digraph"/>  
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    These are synonyms for <Ref Prop="IsReflexiveDigraph"/> and <Ref
+      Prop="IsTransitiveDigraph"/>.  For a preorder digraph (or quasiorder
+    digraph) <A>digraph</A>, the corresponding preorder is the relation
+    <M>\leq</M>, defined by <M>x \leq y</M> if and only if <C>[x, y]</C> is an
+    edge of <A>digraph</A>.
+
+<Example><![CDATA[
+gap> gr := Digraph([[1], [2, 3], [2, 3]]);
+<digraph with 3 vertices, 5 edges>
+gap> IsPreorderDigraph(gr);
+true
+gap> gr := Digraph([[1 .. 4], [1 .. 4], [1 .. 4], [1 .. 4]]);
+<digraph with 4 vertices, 16 edges>
+gap> IsPreOrderdigraph(gr);
+true
```

Comment:
"IsPreOrderdigraph", I guess you did not run these manual examples because they would've failed. Also, since the travis tests passed, you must have forgotten to actually include the doc in the manual in the appropriate z-whatever file.